_posts/en/posts/2018-06-07-release-0.16.1.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Bitcoin Core 0.16.1 Released
+name: blog-release-0.16.1
+id: en-blog-release-0.16.1
+lang: en
+type: posts
+layout: post
+share: true
+version: 1
+
+excerpt: >
+  Bitcoin Core 0.16.1 has been released with the latest bug fixes and
+  minor updates
+---
+Bitcoin Core version 0.16.1 is now available for [download][download
+page].  All users are encouraged to upgrade to this [maintenance
+release][] that fixes several bugs and provides backports of new minor
+features, such as:
+
+- Mitigating a vector for denial-of-service attacks.  This attack required
+  compromising particular services and would've probably been most
+  effective against nodes that were started for the first time (rather
+  than nodes that had already been connected to the network for more
+  than a few hours).
+
+- Fixing a bug that could've potentially caused miners to lose revenue if
+  they produced two blocks in very rapid succession.
+
+- Ceasing relay of transactions using the rarely-seen `OP_CodeSeparator` opcode for legacy
```

Comment:
Standard style for opcodes is all caps ie `OP_CODESEPERATOR`.

Reply:
@jnewbery I'm aware that's the common practice, and I think it's silly.  Capitalizing everything makes both reading and writing harder for no good reason that I can see.

But if you'd like, I can change it.

Also, this absolutely should not be merged until the final binaries are in /bin, or all of the download links on the site will be broken.

Reply:
Agree with @jnewbery, best to just stick to the convention: https://www.google.com/search?q=OP_CODESEPERATOR

Reply:
Changed to uppercase.  Commits and preview updated.  (But I continue to think we're painting the bikeshed an ugly color :-)  (Also, for some inexplicable reason I get two orders of magnitude more results on Google for the upper camel case version: https://www.google.com/search?q=OP_CodeSeparator )

Reply:
> Also, for some inexplicable reason I get two orders of magnitude more results on Google for the upper camel case version

The heuristic they use to determine the number of results is highly experimental ;)